td-shim/src/lib.rs

```diff
@@ -6,21 +6,55 @@
 
 use scroll::{Pread, Pwrite};
 
-use td_uefi_pi::pi;
+use td_uefi_pi::{pi, pi::guid};
```

Comment:
You may also use use td_uefi_pi::pi::{self, guid};

Reply:
Thanks for reminding me